### ISSUE TITLE
Test on my data

### ISSUE THREAD

**Issue by LitterWindwind** [Karel911/TRACER#17]
Thanks for your great work! How can I use the trained model to test when I got the best model after training on my data？ Because it seems that the pre training model is still used in the inference stage?

**Comment by Karel911** [comment]
Hi,
Thank you for your interest in our research.

If you trained TRACER with your own dataset, the model was saved to the `results/your_own_dataset_name/TEX_0/best_model.pth` (maybe you set the default exp_num).

If you had test GTs, just build with `python main.py test --exp_num 0 --arch 0 --img_size 320`.
Or not, build with `python main.py inference --dataset custom_dataset/ --exp_num 0 --arch x --img_size xxx --save_map True`.after modifying the line at [inference.py](https://github.com/Karel911/TRACER/blob/main/inference.py#L29) to `path = os.path.join(save_path, 'best_model.pth')`

I guess it works.

**Comment by LitterWindwind** [comment]
THANKS! The number of my own data sets is too small. When the batch size is 32, the Val loss is always 0 (there are only 25 Val pictures). 
I mistook it for the pre-train model that has been used for reasoning since the bad inference results. Now I get it!

**Comment by LitterWindwind** [comment]
When I modify the line at [inference.py](https://github.com/Karel911/TRACER/blob/main/inference.py#L29) to use my model, there seems to be a problem
![image](https://user-images.githubusercontent.com/35981299/168745086-6607ba43-e5f8-4578-b024-890931e6c21b.png)

**Comment by Karel911** [comment]
Please edit the `self.model.load_state_dict(path)` to `self.model.load_state_dict(torch.load(path))`.

**Comment by LitterWindwind** [comment]
Thanks for your patient reply!
